### ISSUE TITLE
New Breakpoints missing withing BD's "Hide on Breakpoint"

### ISSUE THREAD

**Issue by lordloadalot** [soflyy/breakdance-bugs#845]
Hi Team!

I just add a new Breakpoint under "Preferences". My aim was to hide Element on that specific breakpoint.
Unfortunately it doesn't show under "GEAR SYMBOL" >> "HIDE ON BREAKPOINT" >> HIDE ( Choose Breakpoints"

Please help!
Thank You!

![Bildschirmfoto 2023-06-12 um 16 46 05](https://github.com/soflyy/breakdance-bugs/assets/29510055/5a9aa1dc-8c3c-4592-9adc-4e9d5f037ba4)
![Bildschirmfoto 2023-06-12 um 16 45 44](https://github.com/soflyy/breakdance-bugs/assets/29510055/7caa4f46-acec-4135-931c-ea0845661f66)

**Comment by brandforwardnl** [comment]
Yes, I spotted the same bug ;-) Hope that this can be fixed in the near future.

**Comment by lordloadalot** [comment]
I've been told to convert this into a feature request. otherwise it wil be ignored. I think it's a bug - not a missing feature ...

Von meinem iPhone gesendet

**Comment by brandforwardnl** [comment]
This is definately a bug because the 'show on x device' feature is useless if you have custom breakpoints. But I'll make a 'feature request' it it doesn't already exists.

**Comment by lordloadalot** [comment]
Any NEWS here ?

**Comment by louisreingold** [comment]
Comments bumping feature requests without adding additional value, i.e. "any update" are not appropriate or helpful.

If you want to see how popular your request is (which might result it in it being fast tracked), post on https://breakdance.canny.io/ so others can upvote.

This is a bug tracker. We don't prohibit feature requests here (we love the feedback and we'll take it anywhere we can get it), but as a product manager I have to prioritize things based on what thousands of people want. That's why we like the Canny board, where people can vote.

**Comment by louisreingold** [comment]
This isn't particularly simple to implement, because you could easily build a custom media query that wouldn't work with the Hide By Breakpoint feature (read the generate CSS to see what I mean...). Media queries cascade down (when you do it desktop-first like we do), and our feature is Hide By Breakpoint, not Hide At and Below Breakpoint... so the media query pixel values need to line up exactly with the next smallest media query. You can't just make this work with any custom media query someone could write.

My advice would be to simply set `display: none;` in the custom CSS box for the media query you want to hide the element at.

**Comment by louisreingold** [comment]
Maybe we could add the breakpoints to the box but add a warning that the behavior may not be as expected unless the breakpoint is written in a certain way.

**Comment by lordloadalot** [comment]
Hi Louis,

I have difficulties in understanding **"and our feature is Hide By Breakpoint, not Hide At and Below Breakpoint".**
>> Your breakpoint feature says exactly that : "Table Portrait - 1023 px and below" ( means down to the next maximum set )
>> Your interface says: " Maximum width - minimum width". ( I cannot leave the "minimum width" empty ).
<img width="356" alt="Bildschirmfoto 2023-07-10 um 10 35 22" src="https://github.com/soflyy/breakdance-bugs/assets/29510055/16fc2608-1f76-4ef1-9fa4-6713a46cdb57">

Please help me understand.
Best Martin

**Comment by louisreingold** [comment]
Look at the CSS it outputs for hiding by breakpoint. It has to generate a min-width for the media query, otherwise it doesn't work.

You can leave min-width empty unless we changed something very recently that I don't know about.

**Comment by lordloadalot** [comment]
Hi Louis. Okay true. So I will phrase it in another way. Every Text single widget on this screenshot is only visible at the (standard BD - ) breakpoint of the name it is showing ( via integrated BD "Hide on Breakpoint"  ).
BUT: An iPad Pro always outputs the "Tablet Landscape Widget" when in "Portrait"-mode because of its sheer pixel dimensions. So how to show the "iPad Pro /Custom" text widget only on IPad Pro ?
>>>>
when working with { display: none! } on 5 standard Breakpoints it will not appear with { display: unset } or { display:  inline } on the 5th.

Best Martin.

<img width="1406" alt="Bildschirmfoto 2023-07-10 um 12 45 45" src="https://github.com/soflyy/breakdance-bugs/assets/29510055/23f8c0cd-663c-438c-b24b-244bc154b649">

<img width="1440" alt="Bildschirmfoto 2023-07-10 um 13 12 58" src="https://github.com/soflyy/breakdance-bugs/assets/29510055/f5a5bbfb-9c15-4b24-a2a5-fce85bec9da5">

**Comment by MrBushid0** [comment]
+1

I just noticed this as well while working on an extension. It would be great if the breakdown_dropdown control supported custom breakpoints as well. 

Wanted to create a custom one via the normal dropdown control but it's not possible to select multiple entries at once on that one.
